Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM --platform=${BUILDPLATFORM:-linux/amd64} cgr.dev/chainguard/go:1.20 AS builder
+#syntax=docker/dockerfile:1.4
+
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:1.1.0 AS xx
```

Comment:
I'm not super familiar with `tonistiigi/xx` or using it to build cross platform images, so some of these will just be comments/clarification for these changes.

But _looks_ like it's a base image toolset for building cross compatible images and is useful for cross-platform building?

Reply:
> But _looks_ like it's a base image toolset for building cross compatible images and is useful for cross-platform building?

yes